.env.docker

```
@@ -87,6 +87,16 @@ ONETL_SFTP_PORT=2222
 ONETL_SFTP_USER=onetl
 ONETL_SFTP_PASSWORD=AesujeifohgoaCu0Boosiet5aimeitho
 
+# Samba
+ONETL_SAMBA_HOST=localhost
```

```suggestion
ONETL_SAMBA_HOST=samba
```
Имя сервиса из docker-compose